src/utils.ts

```diff
@@ -11,10 +11,40 @@
  * subject to an additional IP rights grant found at
  * http://polymer.github.io/PATENTS.txt
  */
+import * as path from 'path';
 import {parse as parseUrl_, Url} from 'url';
 
 const unspecifiedProtocol = '-:';
 
+export const EOL = /\r\n|\n/g;
+export const LF = '\n';
+
+/**
+ * Replace all line-terminator sequences with newline `\n`.
+ *
+ * @param text - String to convert.
+ */
+export function normalizeNewlines(text: string|string[]): string {
+  if (Array.isArray(text)) {
+    return text.map(normalizeNewlines).join(LF);
+  }
+  return text.replace(EOL, LF);
+}
+
+/**
+ * Ensure path separators used match the platform's separator.
+ * Backslash *is* a valid filename character in a posix
+ * environment, so this function is a no-op there.
+ *
+ * @param pathname - file path to transform.
+ */
+export function normalizePathSeparators(pathname: string): string {
```

Comment:
@rictic had previously suggested "convertToPlatformsPathSeparators" which I think is the least ambiguous.  I'll go with that.  Will come up with something snappier when we make a common package.

Added TODO.